### PR TITLE
Develop layout_product-cards

### PR DESCRIPTION
**Pull request by sergey-lipikhin**
- [DEMO LINK](https://sergey-lipikhin.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://sergey-lipikhin.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+  width: 200px;
+  box-sizing: border-box;
+
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  font-family: Roboto, Helvetica, sans-serif;
```
Comment (MaksymMaliuk) [inline src/styles/blocks/card.css:11]: font-family can be placed into reset.css

**Review by MaksymMaliuk — APPROVED** [review]
GJ

File: src/styles/blocks/card.css (hunk 1 above)
Comment (illiarerun) [reply src/styles/blocks/card.css:11]: Nope, move it normalize under `html` selector

File: src/styles/blocks/card.css [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+  color: #060b35;
```
Comment (illiarerun) [inline src/styles/blocks/card.css:12]: use variables for commonly used values, such as colors

**Review by illiarerun — APPROVED** [review]

**Pull request closed without merging**